File: src/main/java/main/pre_SetUp/CreateFolderForUsersAudioInputs.java
package main.pre_SetUp;


import objects.sound_files.SoundFile.Directories;

import java.io.IOException;
import java.nio.file.Files;

public class CreateFolderForUsersAudioInputs implements IPreSetUp {
    @Override
    public void GetReady() {
        try {
            if (!Files.exists(Directories.getthePath())) Files.createDirectory(Directories.getthePath());
            else System.out.println("already exists");
        } catch (IOException e) {
            e.printStackTrace();
        }
    }
}

File: src/main/java/main/pre_SetUp/CreateFolderForUsersAudioInputs.kt
package main.pre_SetUp

import objects.SoundFile.Directories
import java.io.IOException
import java.nio.file.Files

class CreateFolderForUsersAudioInputs : IPreSetUp {
    override fun GetReady() {
        try {
            if (!Files.exists(Directories.getthePath())) Files.createDirectory(Directories.getthePath()) else println("already exists")
        } catch (e: IOException) {
            e.printStackTrace()
        }
    }
}